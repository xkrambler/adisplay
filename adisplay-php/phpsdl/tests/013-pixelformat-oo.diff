002+ Segmentation fault (core dumped)
003+ 
004+ Termsig=11
002- string(15) "SDL_Palette(16)"
003- = Color
004- string(22) "SDL_Color(85,85,85,85)"
005- = PaletteColors
006- int(0)
007- = PixelFormat
008- string(42) "SDL_PixelFormat(SDL_PIXELFORMAT_INDEX4LSB)"
009- = PixelFormatPalette
010- int(0)
011- = Result
012- object(SDL_PixelFormat)#%d (16) {
013-   ["format"]=>
014-   int(303039488)
015-   ["BitsPerPixel"]=>
016-   int(4)
017-   ["BytesPerPixel"]=>
018-   int(1)
019-   ["Rmask"]=>
020-   int(0)
021-   ["Gmask"]=>
022-   int(0)
023-   ["Bmask"]=>
024-   int(0)
025-   ["Amask"]=>
026-   int(0)
027-   ["Rloss"]=>
028-   int(8)
029-   ["Gloss"]=>
030-   int(8)
031-   ["Bloss"]=>
032-   int(8)
033-   ["Aloss"]=>
034-   int(8)
035-   ["Rshift"]=>
036-   int(0)
037-   ["Gshift"]=>
038-   int(0)
039-   ["Bshift"]=>
040-   int(0)
041-   ["Ashift"]=>
042-   int(0)
043-   ["palette"]=>
044-   object(SDL_Palette)#%d (4) {
045-     ["ncolors"]=>
046-     int(16)
047-     ["version"]=>
048-     int(2)
049-     ["refcount"]=>
050-     int(2)
051-     ["colors"]=>
052-     array(16) {
053-       [0]=>
054-       object(SDL_Color)#%d (4) {
055-         ["r"]=>
056-         int(0)
057-         ["g"]=>
058-         int(0)
059-         ["b"]=>
060-         int(0)
061-         ["a"]=>
062-         int(0)
063-       }
064-       [1]=>
065-       object(SDL_Color)#%d (4) {
066-         ["r"]=>
067-         int(17)
068-         ["g"]=>
069-         int(17)
070-         ["b"]=>
071-         int(17)
072-         ["a"]=>
073-         int(17)
074-       }
075-       [2]=>
076-       object(SDL_Color)#%d (4) {
077-         ["r"]=>
078-         int(34)
079-         ["g"]=>
080-         int(34)
081-         ["b"]=>
082-         int(34)
083-         ["a"]=>
084-         int(34)
085-       }
086-       [3]=>
087-       object(SDL_Color)#%d (4) {
088-         ["r"]=>
089-         int(51)
090-         ["g"]=>
091-         int(51)
092-         ["b"]=>
093-         int(51)
094-         ["a"]=>
095-         int(51)
096-       }
097-       [4]=>
098-       object(SDL_Color)#%d (4) {
099-         ["r"]=>
100-         int(68)
101-         ["g"]=>
102-         int(68)
103-         ["b"]=>
104-         int(68)
105-         ["a"]=>
106-         int(68)
107-       }
108-       [5]=>
109-       object(SDL_Color)#%d (4) {
110-         ["r"]=>
111-         int(85)
112-         ["g"]=>
113-         int(85)
114-         ["b"]=>
115-         int(85)
116-         ["a"]=>
117-         int(85)
118-       }
119-       [6]=>
120-       object(SDL_Color)#%d (4) {
121-         ["r"]=>
122-         int(102)
123-         ["g"]=>
124-         int(102)
125-         ["b"]=>
126-         int(102)
127-         ["a"]=>
128-         int(102)
129-       }
130-       [7]=>
131-       object(SDL_Color)#%d (4) {
132-         ["r"]=>
133-         int(119)
134-         ["g"]=>
135-         int(119)
136-         ["b"]=>
137-         int(119)
138-         ["a"]=>
139-         int(119)
140-       }
141-       [8]=>
142-       object(SDL_Color)#%d (4) {
143-         ["r"]=>
144-         int(136)
145-         ["g"]=>
146-         int(136)
147-         ["b"]=>
148-         int(136)
149-         ["a"]=>
150-         int(136)
151-       }
152-       [9]=>
153-       object(SDL_Color)#%d (4) {
154-         ["r"]=>
155-         int(153)
156-         ["g"]=>
157-         int(153)
158-         ["b"]=>
159-         int(153)
160-         ["a"]=>
161-         int(153)
162-       }
163-       [10]=>
164-       object(SDL_Color)#%d (4) {
165-         ["r"]=>
166-         int(170)
167-         ["g"]=>
168-         int(170)
169-         ["b"]=>
170-         int(170)
171-         ["a"]=>
172-         int(170)
173-       }
174-       [11]=>
175-       object(SDL_Color)#%d (4) {
176-         ["r"]=>
177-         int(187)
178-         ["g"]=>
179-         int(187)
180-         ["b"]=>
181-         int(187)
182-         ["a"]=>
183-         int(187)
184-       }
185-       [12]=>
186-       object(SDL_Color)#%d (4) {
187-         ["r"]=>
188-         int(204)
189-         ["g"]=>
190-         int(204)
191-         ["b"]=>
192-         int(204)
193-         ["a"]=>
194-         int(204)
195-       }
196-       [13]=>
197-       object(SDL_Color)#%d (4) {
198-         ["r"]=>
199-         int(221)
200-         ["g"]=>
201-         int(221)
202-         ["b"]=>
203-         int(221)
204-         ["a"]=>
205-         int(221)
206-       }
207-       [14]=>
208-       object(SDL_Color)#%d (4) {
209-         ["r"]=>
210-         int(238)
211-         ["g"]=>
212-         int(238)
213-         ["b"]=>
214-         int(238)
215-         ["a"]=>
216-         int(238)
217-       }
218-       [15]=>
219-       object(SDL_Color)#%d (4) {
220-         ["r"]=>
221-         int(255)
222-         ["g"]=>
223-         int(255)
224-         ["b"]=>
225-         int(255)
226-         ["a"]=>
227-         int(255)
228-       }
229-     }
230-   }
231- }
232- = RGB
233- int(8)
234- int(12)
235- NULL
236- int(85)
237- int(85)
238- int(85)
239- NULL
240- int(170)
241- int(170)
242- int(170)
243- int(170)
244- = Done