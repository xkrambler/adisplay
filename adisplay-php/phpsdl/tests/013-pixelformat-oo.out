= Palette
Segmentation fault (core dumped)

Termsig=11